ObjCVideoQuickstart/ViewController.m

```diff
@@ -142,10 +142,10 @@ - (void)startPreview {
 }
 
 - (void)flipCamera {
-    if (self.camera.source == TVIVideoCaptureSourceFrontCamera) {
-        [self.camera selectSource:TVIVideoCaptureSourceBackCameraWide];
+    if (self.camera.source == TVICameraCaptureSourceFrontCamera) {
+        [self.camera selectSource:TVICameraCaptureSourceBackCameraWide];
     } else {
-        [self.camera selectSource:TVIVideoCaptureSourceFrontCamera];
+        [self.camera selectSource:TVICameraCaptureSourceFrontCamera];
     }
 }
```

Comment:
Adjust mirroring of TVIVideoView here, or in the TVICameraCapturerDelegate callback?